extensions/azurePublish/src/components/ChooseProvisionAction.tsx

```diff
@@ -100,40 +100,48 @@ const CreateActionContent = () => {
         <Instruction>
           <InstructionTitle>{formatMessage('Step 2')}</InstructionTitle>
           <InstructionDetails>
-            <Text>
-              {formatMessage(
-                'Select tenant and subscription, enter resource group name and resource name, and select region.'
-              )}
-            </Text>
+            <Text>{formatMessage('Select Azure subscription and resource group name')}</Text>
           </InstructionDetails>
         </Instruction>
         <Instruction>
           <InstructionTitle>{formatMessage('Step 3')}</InstructionTitle>
           <InstructionDetails>
-            <Text>
-              {formatMessage(
-                'Review and create new resources. Once provisioned these resources will be available in your Azure portal.'
-              )}
-            </Text>
+            <Text>{formatMessage('Select and name your new resources')}</Text>
+          </InstructionDetails>
+        </Instruction>
+        <Instruction>
+          <InstructionTitle>{formatMessage('Step 4')}</InstructionTitle>
+          <InstructionDetails>
+            <Text>{formatMessage('Review and confirm resources to be created')}</Text>
           </InstructionDetails>
         </Instruction>
       </Details>
+      <Summary>
+        <Text>{formatMessage('Once provisioned, your new resources will be available in the Azure portal. ')}</Text>
+        <LearnMoreLink href="https://aka.ms/composer-publish-bot#create-new-azure-resources" target="_blank">
+          {formatMessage('Learn More')}
+        </LearnMoreLink>
+      </Summary>
     </Content>
   );
 };
 
 const ImportActionContent = () => {
   return (
     <Content>
-      <Title>{formatMessage('Import existing resources')}</Title>
+      <Title>{formatMessage('Use existing resources')}</Title>
       <Summary>
         <p>
-          <Text>{formatMessage('Select this option to import existing Azure resources and publish a bot.')}</Text>
+          <Text>
+            {formatMessage(
+              'Select this option if you have access to existing Azure resources and their associated values.'
+            )}
+          </Text>
         </p>
         <p>
           <Text>
             {formatMessage(
-              'Edit the JSON file in the Publish Configuration field. You will need to find the values of associated resources in your Azure portal. A list of required and optional resources may include:'
+              'Copy and paste the JSON file containing the values of your existing Azure resources, from the Azure portal. This file includes values for some or all of the following:'
```

Comment:
Grammatically, that comma is incorrect. 
```suggestion
              'Copy and paste the JSON file containing the values of your existing Azure resources from the Azure portal. This file includes values for some or all of the following:'
```

Reply:
Whatever Marieke wants, Marieke gets :)

Reply:
@mareekuh can you comment here?